compiler/lib/id.ml

```diff
@@ -0,0 +1,39 @@
+(* Js_of_ocaml compiler
+ * http://www.ocsigen.org/js_of_ocaml/
+ * Copyright (C) 2010 Jérôme Vouillon
+ * Laboratoire PPS - CNRS Université Paris Diderot
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, with linking exception;
+ * either version 2.1 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA 02111-1307, USA.
+ *)
+type identifier = string
+
+type ident_string =
+  { name : identifier
+  ; var : Code.Var.t option
+  ; loc : Loc.t
+  }
+
+type t =
+  | S of ident_string
+  | V of Code.Var.t
+
+let ident =
+  (fun ?(loc = N) ?var name -> (S { name; var; loc } [@explicit_arity])
+    : ?loc:Loc.t -> ?var:Code.Var.t -> identifier -> t)
+
+type property_name =
```

Comment:
`property_name` is very much specific to javascript, I don't think we need to use it in the new IR

Reply:
Indeed, removed.